manager/integration/tests/test_basic.py

```diff
@@ -2195,7 +2195,8 @@ def test_engine_image_daemonset_restart(client, apps_api, volume_name):  # NOQA
     check_volume_data(volume, snap1_data)
 
     # Wait for the restart complete
-    common.wait_for_engine_image_state(client, default_img.name, "ready")
+    ei_state = common.get_engine_image_status_value(client, default_img.name)
+    common.wait_for_engine_image_state(client, default_img.name, ei_state)
```

Comment:
This logic has some problem: query the current state of the engine image, then compare the value by querying the engine image state again.

In 1.1.1, I think we should just check the state == `deployed`?

Reply:
After @c3y1huang explained, I think this is not an issue.